<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CremittancePercentSetup</class>
 <widget class="QDialog" name="CremittancePercentSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="m_percentTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>721</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Account Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Local Share</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>HQ Share</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>311</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="m_deptLbl">
      <property name="text">
       <string>Department</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_deptComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>291</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="m_accountLbl">
      <property name="text">
       <string>Account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_accountComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>154</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="m_cancelBtn">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="m_okBtn">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>311</width>
     <height>47</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="m_remitNameLbl">
      <property name="text">
       <string>Remittance Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_remitName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
